README.md

```diff
@@ -56,6 +56,49 @@ module "spacelift_workerpool" {
 }
 ```
 
+### FedRAMP Example
+
+If your account is in the Spacelift FedRAMP environment, set the `is_fedramp` variable to `true`:
+
+```hcl
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-west-1"
+}
+
+module "spacelift_workerpool" {
+  source = "github.com/spacelift-io/terraform-aws-spacelift-workerpool-on-ec2?ref=v4.4.3"
```

Comment:
This should be updated to use the upcoming `5.3.0` tag. The main example should be updated too.